src/stories/button/Buttons.stories.js

```diff
@@ -0,0 +1,94 @@
+
+import React from "react";
+import { FaSave } from "react-icons/fa";
+import Button from "./Buttons";
+import { StyledContainer } from "./Buttons.style";
+
+
+export default {
+  title: "Buttons",
+  component: Button,
+};
+
+
+export const listButton = () => (
+  <StyledContainer>
+    <Button type="primary" variant="contained" text="PRIMARY BUTTON" disableHover />
+    <Button type=" iconAfterLinkColor" variant="contained" text="SECONDARY BUTTON" disableHover  />
```

Comment:
can you please explain this type `iconAfterLinkColor`?

Reply:
Sir it's the color name of button which i added in theme file